modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ClickGoalPicker/ClickGoalPicker.es.js

```diff
@@ -536,7 +546,7 @@ function TargetTopper({allowEdit, geometry, isEditing, selector}) {
 			}}
 		>
 			<span className="mr-2 text-truncate">
-				{isEditing ? selector : Liferay.Language.get('target')}
+				{isEditing ? `#${selector}` : Liferay.Language.get('target')}
```

Comment:
The fact that you have to do this `#${selector}` in a few places is a hint that the variable is named wrong: a word corresponding to an id is not in itself a "selector" until you turn it into one by prepending `#`. Maybe you should be assigning:

```js
const selector = `#${selector}`;
```

somewhere up in the call stack so that the variable really contains what its name implies (either that, or rename the variable).

Reply:
👍